channel/state/signedstate.go

```diff
@@ -92,3 +93,37 @@ func (ss SignedState) Equal(ss2 SignedState) bool {
 	}
 	return true
 }
+
+// MarshalJSON marshals the SignedState into JSON, implementing the Marshaler interface.
+func (ss SignedState) MarshalJSON() ([]byte, error) {
+
+	// Similar type but with public members:
+	type SignedState struct {
```

Comment:
We could use use an anonymous struct so we don't have to declare a new SignedState struct. Not sure if that would be cleaner or not.

Reply:
13593c3